both/accounts/config.js

```diff
@@ -1,3 +1,31 @@
+var pwd = AccountsTemplates.removeField('password');
```

Comment:
https://github.com/meteor-useraccounts/core/blob/master/Guide.md#login-with-username-or-email

Referencing for future comment